### ISSUE TITLE
Feature Request: zoom to coordinate feature

### ISSUE THREAD

**Issue by celmi** [opengisch/qgis-swiss-locator#58]
Just an idea (because I intuitively thought that this would work): it would be nice if I could copy paste coordinates into the search window and this place would then be shown (especially such that are formatted in the typical swiss format, e.g. 2'672'787/1'252'535)

**Comment by celmi** [comment]
I'm sorry, I don't understand your answer. What is the go native locator filter?

**Comment by 3nids** [comment]
<img width="748" alt="image" src="https://github.com/opengisch/qgis-swiss-locator/assets/127259/37868e2c-73b6-42d4-a0fb-df44891b4b4a">

**Comment by celmi** [comment]
ok, I'm getting it now, thanks!
But this only works when coordinates are completely clean. This here
2'672'787 1'252'535
will not accepted, because it contains hyphens. 
I need to manually remove the hyphens first. Ironically, the swiss-locator dialogue then present's the same coordinates anew with hyphens
![go](https://github.com/opengisch/qgis-swiss-locator/assets/21191672/d484fcc4-26af-4ac5-b356-5e1b5f954a39)

I think in the typical use case where one / I would want to copy-paste coordinates from somehwere, those coordinates are formatted with hyphens for better legility. It would be nice to be able to use the "go" locator with strings containig such formatted coordinates.

**Comment by celmi** [comment]
I guess the title of the feature request should be changed to "allow formatted strings for coordinate input" ?

**Comment by 3nids** [comment]
this would be a feature request / bug report in QGIS.
I suspect this is depending on the locale settings you're using, hyphens are not always used as thousand separators.
